src/integration/java/io/pinecone/helpers/GetOrCreateIndex.java

```diff
@@ -0,0 +1,39 @@
+package io.pinecone.helpers;
+
+import io.pinecone.*;
+import io.pinecone.model.CreateIndexRequest;
+import io.pinecone.model.IndexMeta;
+
+import java.io.IOException;
+import java.util.List;
+
+public class GetOrCreateIndex {
+    public static PineconeConnection getOrCreateIndex(int dimension) throws IOException {
```

Comment:
Could this helper function also be reused in the `ConfigureIndexTest` file?

Reply:
For any integration test, we would like to make the sub test independent and not reply on the previous sub test. To achieve this, we are going to make use of BeforeEach and setting a unique namespace in data plane but for configure index, we must create a new index.